bigdl/README.md

```diff
@@ -10,11 +10,16 @@ More information [project's website](https://analytics-zoo.github.io)
 
 ## Using this initialization action
 
+**:warning: NOTICE:** See [best practices](/README.md#how-initialization-actions-are-used) of using initialization actions in production.
+
 You can use this initialization to create a new Dataproc cluster with BigDL's Spark and PySpark libraries installed.
 
 ```
-gcloud dataproc clusters create <CLUSTER_NAME> \
-    --initialization-actions gs://$MY_BUCKET/bigdl/bigdl.sh \
+REGION=<region>
+CLUSTER_NAME=<cluster_name>
+gcloud dataproc clusters create ${CLUSTER_NAME} \
+    --region ${REGION} \
+    --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/bigdl/bigdl.sh \
     --initialization-action-timeout 10m
```

Comment:
Should we call out why the 10m timeout? It seems to be more than just an example of specifying a timeout

Reply:
Added explanation at the top of the page.